src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
```

Comment:
Consider adding a proper font https://fonts.google.com/specimen/Roboto 
<img width="1632" alt="Screenshot 2023-10-01 at 18 00 14" src="https://github.com/mate-academy/layout_product-cards/assets/135232001/71062add-df91-422e-975b-4e6ff43043a5">
1. Click on the needed font weights 2. Copy the font link 3. Add the font in the head of your HTML before stylesheets